### PR TITLE
MONGOID-5614 Document ffi dependency

### PR DESCRIPTION
**Pull request by comandeo**

### CODE REVIEW

File: docs/tutorials/automatic-encryption.txt [hunk 1]
```diff
@@ -97,6 +98,17 @@ need to be installed manually. ``mongocryptd`` comes pre-packaged with
 enterprise builds of the MongoDB server (versions 4.2 and newer).
 For installation instructions, see the `MongoDB manual <https://mongodb.com/docs/manual/reference/security-client-side-encryption-appendix/#installation>`_.
 
+Add ``ffi`` to your Gemfile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mongo Ruby driver uses `ffi gem <https://github.com/ffi/ffi>`_ to call
```
Comment (jamis) [inline docs/tutorials/automatic-encryption.txt:104]: Looks like a stray underscore character here. Also, probably only need backticks around the word "ffi" itself.

Also, grammatically: I'd insert "the" before "Mongo" and after "uses" (e.g. "**The** Mongo Ruby driver uses **the** ffi gem...").

**Review by jamis — APPROVED** [review]
Just a formatting issue, and a grammar issue, but looks good otherwise!

**Review by alexbevi — APPROVED** [review]

File: docs/tutorials/automatic-encryption.txt (hunk 1 above)
Comment (jamis) [reply docs/tutorials/automatic-encryption.txt]: Ah! Right. I keep forgetting about that.

Comment (comandeo) [reply docs/tutorials/automatic-encryption.txt]: Thank you for the cleanup!

**Pull request merged**